packages/lesswrong/components/posts/PostsPage/BestOfLessWrong/BestOfLessWrongAnnouncement.tsx

```diff
@@ -0,0 +1,269 @@
+    overflow: "hidden",
+    position: "relative",
+  },
+  categoryImageHover: {
+    opacity: 0,
+  },
+  category: {
+    width: "calc(16% - 2px)",
+    // height: 362,
+    overflow: "hidden",
+    marginBottom: 8,
+    position: "relative",
+    cursor: "pointer",
+    '&:hover': {
+      opacity: 1
+    },
+  },
+  categoryTitle: {
+    ...theme.typography.body1,
+    fontSize: 18,
+    ...theme.typography.headerStyle,
+    // color: theme.palette.grey[700],
+    lineHeight: '1.2',
+    textAlign: 'center',
+    fontWeight: 600,
+    marginTop: 12,
+    position: "absolute",
+    top: "48%",
+    left: "50%",
+    transform: "translate(-50%, -50%)",
+    color: theme.palette.text.alwaysWhite,
+    whiteSpace: 'nowrap',
+    backdropFilter: 'blur(1px)',
+    textShadow: `0 0 4px ${theme.palette.greyAlpha(.8)}, 0 0 8px ${theme.palette.greyAlpha(.2)}`,
+  },
+  winnerItem: {
+    width: "100%",
+    height: 120,
+    overflow: "hidden",
+    borderBottom: theme.palette.border.faint,
+    position: "relative",
+    cursor: "pointer",
+    marginBottom: 1,
+    zIndex: 1,
+    '&:hover $winnerImage': {
+      opacity: 1,
+      filter: "brightness(1) saturate(1)",
+    },
+    '&:hover $winnerCategoryRank': {
+      color: theme.palette.grey[300],
+    },
+    '&:hover $winnerImageBackground': {
+      opacity: .2,
+    },
+  },
+  winnerImage: {
+    width: "100%",
+    position: "absolute",
+    top: 0,
+    left: 0,
+    height: 360,
+    overflow: "hidden",
+    opacity: 0,
+    objectFit: "cover",
+    objectPosition: "center",
+    transform: "scale(2)",
+    // filter: "brightness(0.45) saturate(.5)",
+    // filter: "brightness(.5) saturate(.0)",
+    borderRadius: 2,
+    filter: "brightness(0.75) saturate(.75)",
+    transition: "opacity 0.2s ease-in-out",
+    zIndex: 1,
+  },
+  winnerImageHover: {
+    opacity: 1
+  },
+  winnerTitle: {
+    ...theme.typography.body2,
+    transition: "opacity 0.2s ease-in-out",
+    fontSize: 13,
+    color: theme.palette.grey[300],
+    lineHeight: '1.2',
+    position: "absolute",
+    textShadow: `0px 0px 3px ${theme.palette.text.alwaysBlack}, 0 0 5px ${theme.palette.text.alwaysBlack}`,
+    top: 8,
+    left: 8,
+    right: 8,
+    opacity: 0,
+  },
+  winnerCategoryRank: {
+    ...theme.typography.body2,
+    transition: "opacity 0.1s ease-in-out",
+    fontSize: '.9rem',
+    color: theme.palette.grey[500],
+    lineHeight: '1.2',
+    position: "absolute",
+    bottom: 8,
+    left: 8,
+    opacity: 0,
+    textShadow: `0 0 4px ${theme.palette.greyAlpha(.8)}`,
+  },
+  winnerImageBackground: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    width: "100%",
+    height: "100%",
+    backgroundColor: theme.palette.background.paper,
+    opacity: 0,
+    zIndex: 1,
+  }
+}));
+
+const BestOfLessWrongAnnouncement = () => {
+  const classes = useStyles(styles);
+
+  const { results } = useMulti({
+    terms: {
+      view: "bestOfLessWrongAnnouncement",
+    },
+    limit: 50,
```

Comment:
does this need to be 50?

Reply:
actually even at 18 I am kinda sketched out by the idea of loading 18 posts on homepage SSRs.